src/function/scalar/string/concat.cpp

```diff
@@ -104,13 +104,26 @@ static void concat_operator(DataChunk &args, ExpressionState &state, Vector &res
 		    auto b_data = b.GetData();
 		    auto a_length = a.GetSize();
 		    auto b_length = b.GetSize();
+		    bool is_hexa = (a_length >= 2 && a_data[0] == '\\' && a_data[1] == 'x' &&
```

Comment:
The current concat_operator implementation already supports blobs fully as it is just using memcpy, no? Please revert the concat operator to the previous version.